src/masterless.py

```diff
@@ -114,8 +114,11 @@ def launch(pname, instance_id=None, alt_config='standalone', *repolist):
 @requires_masterless
 def set_versions(stackname, *repolist):
     "call with formula name and a revision, like: builder-private@ab87af78asdf2321431f31"
-    ctx = context_handler.load_context(stackname)
-    repolist = parse_validate_repolist(ctx['project'], *repolist)
+    #ctx = context_handler.load_context(stackname)
+    #repolist = parse_validate_repolist(ctx['project'], *repolist)
+
+    pdata = core.project_data_from_stackname(stackname)
```

Comment:
uses `elife.yaml` data rather than an existing context for an older instance, but for `masterless` usage this is fine